src/Pages/AuthPages/ForceLoginAdminHandler.js

```diff
@@ -34,7 +33,7 @@ const ForceLoginAdminHandler = () => {
         window.location.replace('/');
       }, 1000);
     }
-  }, [navigate, dispatch]);
+  }, [navigate, props]);
```

Comment:
@DevIsraelHub this props is not needed here, remove it and test that everything works as expected